SR2020/2020-06-18-version-0-2-0-simulator.md

```diff
@@ -0,0 +1,23 @@
+---
+to: sr2020-teams
+subject: Virtual Competition Simulation Version 2 Released
+---
+
+Hi
+
+We have released version 0.2.0 of the competition simulation which you can download from the [Docs page](https://studentrobotics.org/docs/competition-simulator/).
+
+The following changes have been made:
+
+- Reducing the range of randomness which can be exhibited by motors and sensors
+- Adding multiple robots to the arena with the ability to assign distinct code files to each
+- Fix the gripper jumping open bug
+- Add a sleep method which runs on "simulation time" instead of actual time. This is useful for computers where the simulation runs slowly
```

Comment:
Have we actually documented this anywhere? If not, that's a blocker for release.

Reply:
Tracking as https://github.com/srobo/competition-simulator/issues/141